voidcrew/code/modules/vending/security.dm

```diff
@@ -0,0 +1,111 @@
+/obj/machinery/vending/security/attackby(obj/item/item, mob/user, params)
+	if(istype(item, /obj/item/gun_voucher))
+		RedeemVoucher(item, user)
```

Comment:
we should just remove gun vouchers, sec (and the bead shuttle?) spawn with one and that's about it. Seems more like some whiteship feature that was ported over into shiptest despite not making any sense.